### DISCUSSION TITLE
Tweedie Distribution

### DISCUSSION THREAD

**Discussion by StatMixedML** [StatMixedML/XGBoostLSS#28] [Q&A]
Dear Community, 

I am receiving constants requests to make the Tweedie distribution available. I would be very happy to do that. The only obstacle is that I cannot find a proper implementation that allows to model mean, power and scale parameters of the distribution. Also, we need to make sure that the implementation allows to draw random samples and to calculate the inverse CDF / quantile function.

Does anyone know a good implementation, preferably in PyTorch?

**Comment by StatMixedML** [comment]
Python Implementation: https://github.com/thequackdaddy/tweedie
PyTorch Loss: https://pytorch-forecasting.readthedocs.io/en/stable/_modules/pytorch_forecasting/metrics/point.html#TweedieLoss

**Comment by sidhenriksen** [comment]
Has there been any development on this? Would love to see Tweedie included in the package.

**Reply by StatMixedML** [reply]
@sidhenriksen Thanks for your interest in the project. Unfortunately, there hasn't been progress on this. Happy if you pick it up!
